battle.go

```diff
@@ -154,6 +155,9 @@ func (b *Battle) SimulateRound() ([]Transaction, bool) {
 		ctxA := turns[i].Context
 		ctxB := turns[j].Context
 		if reflect.TypeOf(turnA) == reflect.TypeOf(turnB) {
+			if ctxA.Pokemon.HeldItem == ItemQuickClaw {
+				return b.rng.Roll(3, 16) // 3/16 chance to move first
```

Comment:
This won't work correctly because the RNG is being rolled multiple times.

Reply:
Going to move this into a separate issue